modules/account-lib/src/coin/stx/walletInitializationBuilder.ts

```diff
@@ -0,0 +1,26 @@
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import { BaseKey } from '../baseCoin/iface';
+import { NotImplementedError } from '../baseCoin/errors';
+import { Transaction } from './transaction';
+import { TransactionBuilder } from './transactionBuilder';
+
+export class WalletInitializationBuilder extends TransactionBuilder {
```

Comment:
There are no wallet initializations in Stacks - this should be removable

Reply:
Removed